hazelcast/src/main/java/com/hazelcast/map/impl/operation/AbstractMultipleEntryOperation.java

```diff
@@ -67,6 +67,15 @@ protected AbstractMultipleEntryOperation(String name, EntryBackupProcessor backu
         this.backupProcessor = backupProcessor;
     }
 
+
+    @Override
+    public void beforeRun() throws Exception {
+        super.beforeRun();
+        if (mapContainer.isWanReplicationEnabled()) {
+            mapContainer.checkWanReplicationQueues();
+        }
```

Comment:
We have a [MapOperation] (https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapOperation.java#L52) to centralize this kind of checks.